textproc/ripgrep/Portfile

```diff
@@ -5,11 +5,14 @@ PortGroup           github 1.0
 PortGroup           cargo 1.0
 
 github.setup        BurntSushi ripgrep 12.1.1
+revision            1
+
 categories          textproc
 platforms           darwin
 maintainers         {raimue @raimue} \
                     openmaintainer
 license             MIT
+supported_archs     x86_64 arm64
```

Comment:
But this removes support for all unlisted archs... That seems like an uninteded side effect?